src/Aura/Signal/Handler.php

```diff
@@ -105,8 +105,10 @@ public function exec($origin, $signal, array $args)
             // match on a specific object
             $match_sender = $this->sender === $origin;
         } else {
-            // match on a class
-            $match_sender = $this->sender == '*' || $origin instanceof $this->sender;
+            // match on a wildcard or an origin
+            $match_sender = $this->sender == '*' || $this->sender == $origin;
+            // match on a parent class
+            $match_sender = $match_sender || $origin instanceof $this->sender;
```

Comment:
can't we do it on a single line ?

Eg : `$match_sender = $this->sender == '*' || $origin instanceof $this->sender || $this->sender == $origin;`

It may be helpful if there is some test case to show how useful it is. Can you add one ?

Thank you.